### PR TITLE
Update common lib and client telemetry

### PR DESCRIPTION
**Pull request by dingmeng-xue**
## Description

It contains couple changes:

- Update common libs to support
  -  Shown correct client request it in debug message #13745
  - Added exception type in client side telemetry 
  - Supported storage API 2019-06-01 #13372
- Update AzModule to support to send correct HTTP status code to telemetry in generated modules. 
- Add client side error when subscription is incorrect in `Get-AzSubscription`

## Checklist

- [ ] I have read the [_Submitting Changes_](../blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/master/CONTRIBUTING.md)
- [ ] The title of the PR is clear and informative
- [ ] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [ ] The PR does not introduce [breaking changes](../blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [ ] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
        - {Please put the link here}
    - [ ] the markdown help files have been regenerated using the commands listed [here](../blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

**Review by isra-fel — APPROVED** [review]
LGTM

File: src/Accounts/Accounts/ChangeLog.md [hunk 1]
```diff
@@ -18,6 +18,9 @@
         - Additional information about change #1
 -->
 ## Upcoming Release
+* Shown correct client request id on debug message [#13745]
```
Comment (isra-fel) [inline src/Accounts/Accounts/ChangeLog.md:21]:
```suggestion
* Showed correct client request id on debug message [#13745]
```
Show / showed / shown

**Review by erich-wang — APPROVED** [review]

File: src/Accounts/Accounts/ChangeLog.md (hunk 1 above)
Comment (dingmeng-xue) [reply src/Accounts/Accounts/ChangeLog.md:21]: shown can be past tense too.

**Pull request merged**